api/v1beta1/common_types.go

```diff
@@ -30,6 +30,11 @@ const (
 	ManilaSchedulerContainerImage = "quay.io/podified-antelope-centos9/openstack-manila-scheduler:current-podified"
 	// ManilaShareContainerImage is the fall-back container image for ManilaShare
 	ManilaShareContainerImage = "quay.io/podified-antelope-centos9/openstack-manila-share:current-podified"
+	//DBPurgeDefaultAge -
```

Comment:
oh, and I think this list of constants might be better located in manila_types.go (they're not common to call controllers, just to top manila controller)

Reply:
Yeah, we only have a top-level webhook enforcing the defaults. I agree it makes sense to define them in `manila_types` instead of common, especially because both the `containerImage` and the `dbPurge` fields are top-level fields (and `containerImage` is inherited in the `subCR` controllers).
+1 on this one